packages/shared/src/components/feeds/FeedCustomPreview.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactElement } from 'react';
+import { useAuthContext } from '../../contexts/AuthContext';
+import { PREVIEW_FEED_QUERY } from '../../graphql/feed';
+import useFeedSettings from '../../hooks/useFeedSettings';
+import { OtherFeedPage, RequestKey } from '../../lib/query';
+import Feed from '../Feed';
+import FeedLayout from '../FeedLayout';
+
+export type FeedCustomPreviewProps = {
+  feedId: string;
+};
+
+export const FeedCustomPreview = ({
+  feedId,
+}: FeedCustomPreviewProps): ReactElement => {
+  const { user } = useAuthContext();
+  const { feedSettings } = useFeedSettings({ feedId });
+  const { advancedSettings, ...previewFilters } = feedSettings || {};
+
+  const feedProps = {
+    feedName: OtherFeedPage.Preview,
+    feedQueryKey: [
+      RequestKey.FeedPreview,
+      user?.id,
+      RequestKey.FeedPreviewCustom,
+      previewFilters,
+    ],
+    query: PREVIEW_FEED_QUERY,
+    forceCardMode: true,
```

Comment:
This just forces that we always use cards for previews, we use it on onboarding as well? If something changed with list mode I'd be happy to update? 🙏

Reply:
So the new list mode (only if users are in the new list mode experiment with the new layout) does not consider this and if the user has list mode selected, the preview of their feed will show on list mode not just cards. 

We added list mode in a lot of places that didn't have list mode before, squads as well. But if this definitely needs to be taken in consideration and preview should always show as cards then we'll need to adjust what we have now.

Reply:
Got it!

Reply:
I think at the time of onboarding it was decided to always force card mode indeed.
Not sure we eventually got designs for preview feed in list mode?

Maybe we can raise it and fix later, won't block release imo.

Reply:
on onboarding it will always be cards anyways as cards are by default, in these other previews might make sense to have card or list depending on user settings.